eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -63,9 +63,14 @@ jobs:
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
         parameters:
           NuGetCacheKey: Test
+      - script: >-
+          dotnet tool install dotnet-dump --global
```

Comment:
Trying to install this tool, which is apparently required to use the --blame-test-crash flag for dotnet test.
I keep getting this message:
"Specify which project file to use because this '/mnt/vss/_work/1/s' contains more than one project file."

Reply:
adding working directory seems to have fixed this.